lib/msf/core/exploit/remote/kerberos/ticket/storage/base.rb

```diff
@@ -102,7 +118,15 @@ def objects(options, &block)
       unless (info = loot_info(options)).blank?
         filter[:info] = info
       end
-      framework.db.loots(workspace: myworkspace, ltype: 'mit.kerberos.ccache', **filter).each do |stored_loot|
+      loots = framework.db.loots(workspace: myworkspace, ltype: 'mit.kerberos.ccache', **filter).select do |loot|
```

Comment:
Let's wrap this up in a try/catch for `ActiveRecord::RecordNotFound` and raise a ticket storage specific exception that clients can handle correctly

Reply:
We'll also have to catch the leaky details of the remote db http service